eng/common/performance/perfhelixpublish.proj

```diff
@@ -152,6 +177,11 @@
       <PayloadDirectory>$(WorkItemDirectory)\ScenarioCorrelation</PayloadDirectory>
       <Command>$(Python) %HELIX_CORRELATION_PAYLOAD%\performance\src\scenarios\crossgen2\test.py crossgen2 --single System.Private.CoreLib.dll --core-root %HELIX_CORRELATION_PAYLOAD%\Core_Root</Command>
     </HelixWorkItem>
+    <HelixWorkItem Include="Crossgen2 Composite Framework R2R">
+      <PayloadDirectory>$(WorkItemDirectory)\ScenarioCorrelation</PayloadDirectory>	
+      <Command>$(Python) %HELIX_CORRELATION_PAYLOAD%\performance\src\scenarios\crossgen2\test.py crossgen2 --composite %HELIX_CORRELATION_PAYLOAD%\performance\src\scenarios\crossgen2\framework-r2r.dll.rsp --core-root %HELIX_CORRELATION_PAYLOAD%\Core_Root</Command>	
+      <Timeout>1:00</Timeout>  
```

Comment:
Why 1 hour timeout? This leads to very long hangs blocking a machine for long periods of time.

I also see the other workitems we send have 4 hours timeouts. Can we modify this for the purpose of PR runs?

Reply:
For crossgen2 composite workitem, 1 hour timeout also takes care of CI runs where we run 5 iterations, while we only have 1 iteration for PR runs and each iteration takes ~8min. For the rest of the workitems we will be able to shorten the timeout once the partition is enabled. @DrewScoggins  Do you have an estimate of the timeout value after partition?

Reply:
I opened: https://github.com/dotnet/runtime/issues/38136 to follow up.

The reason why this concerns me is that if we enter a deadlock like: https://github.com/dotnet/runtime/issues/38138 the machine is blocked.

Reply:
For the micro benchmarks we should be able to set the timeout to about 10 minutes once we get this partition change in. A run take about 3-5 minutes, but I want to leave some wiggle room. These are of course Helix timeouts.

Reply:
Yeah, `Helix` timeouts are the ones we care about reducing on PRs for these micro benchmarks. 10-15 mins sounds good to me.